src/Command/InstallRecipesCommand.php

```diff
@@ -36,9 +36,10 @@ public function __construct(/* cannot be type-hinted */ $flex, string $rootDir)
 
     protected function configure()
     {
-        $this->setName('symfony:sync-recipes')
-            ->setAliases(['sync-recipes', 'fix-recipes'])
+        $this->setName('symfony:recipes:install')
+            ->setAliases(['symfony:sync-recipes', 'sync-recipes', 'fix-recipes'])
```

Comment:
Can we also print a deprecation message when one uses an old alias?

Reply:
Yes, I just added it 
I'm not sure about the message, can you tell me if it's good for you ?

Reply:
Also, should we add a "short" alias for `recipes:install` - most other `symfony:` commands have that short alias: https://github.com/symfony/flex/pull/562/files#diff-3354bea129aa2979af719cb4c3662f34R41